app/controllers/trainees/employing_schools_controller.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+module Trainees
+  class EmployingSchoolsController < ApplicationController
+    before_action :authorize_trainee
+    before_action :load_schools
+
+    def index
+      @trainee_employing_school_form = TraineeEmployingSchoolForm.new(trainee)
+    end
+
+    def update
+      @trainee_employing_school_form = TraineeEmployingSchoolForm.new(trainee, params: trainee_params, user: current_user)
+      save_strategy = trainee.draft? ? :save! : :stash
+
+      if @trainee_employing_school_form.public_send(save_strategy)
+        redirect_to trainee_training_details_confirm_path(@trainee_employing_school_form.trainee)
```

Comment:
I didn't notice this in my PR, but we could replace `@trainee_lead_school_form.trainee` with just `trainee`.